requirements.txt

```diff
@@ -11,4 +11,5 @@ seaborn>=0.11.1
 category_encoders>=2.0.0
 statsmodels >= 0.12.2
 imbalanced-learn>=0.8.0
+pmdarima==1.8.0
```

Comment:
Now that we've disabled arima for conda installs while pmdarima sorts out the dependency issue: is there any reason we shouldn't do
```python
pmdarima>=1.8.0
```
here?

Reply:
This unfortunately causes `latest_dependency_versions` to [throw an error](https://github.com/alteryx/evalml/runs/2529399400?check_suite_focus=true) as it expects `1.8.2`, but that version pins `numpy` to a lower version than what we can accept.